daycare/test_image.py

```diff
@@ -3,6 +3,8 @@
 from .models import Image
 from django.core.exceptions import ValidationError
 
+DEFAULT_DAYCARE_PROFILE_URL = "../../static/images/daycare-default-profile-image.jpeg"
+
 
```

Comment:
you can add the `DEFAULT_DAYCARE_PROFILE_URL` to Conftest.py file instead of using a constant multiple times